### DISCUSSION TITLE
Cast screen via USB with Audio

### DISCUSSION THREAD

**Discussion by dijo7** [barry-ran/QtScrcpy#446] [Ideas]
An option to stream audio in pc while disabling in phone.

**Comment by guanzhangrtk** [comment]
There are some limitations depending on Android version, have you tried [sndscpy](https://github.com/rom1v/sndcpy) or Bluetooth?

**Reply by dijo7** [reply]
Yes bluetooth works. Android 11, OxygenOS, sndscpy keeps disconnecting
